### PR TITLE
docs: add mint+burn documentation

### PR DESCRIPTION
**Pull request by cgorenflo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,76 @@ Optional flags:
...
+- Create verification json object for Axelar
+  ```
+  axelarcli q bitcoin txInfo {blockhash, e.g. 4ac9dc50dc1b952cb1efca1e634216da2f5e3a12b4a4a802ce0f6b1271876bd2} {txID:voutIdx, e.g. da5b2e8037ce4b95f40ada01c6c2cd3ccb806d0a952906130eb9b806f7887590:1}
+  -> returns json of verification info for the given outpoint (copy the escaped string)
+  ```
+- Verify the Bitcoin outpoint
+  ```
+  axelarcli tx bitcoin verifyTx {verification info (output of previous cmd), e.g. "{\"OutPoint\":{\"Hash\":\"NxF/hGLGQZ6mTNyMWkYHPJ21E+2PMb1DV/beV7R9Gpk=\",\"Index\":1},\"Amount\":\"100000000\",\"BlockHash\":\"tPqsKekDOp5lW6QUl+YwlaD/3cmbQJwuUqgiNkqloQM=\",\"Address\":\"bcrt1qrnc097fuepeyrchganj4jzl2yuf5c0fg800uenr5h0d58emztxasusnk7p\",\"Confirmations\":\"21\"}"} --from validator -y -b block
+  -> wait for verification to be confirmed (~10 blocks)
+  ```
```
Comment (jcs47) [inline README.md:46]: Warn about the need to wait for the verification to be voted

**Review by sergeynog — APPROVED** [review]
I'll approve and will just make a few edits later.

File: README.md [hunk 2]
```diff
@@ -21,3 +21,76 @@ Optional flags:
...
+- Trigger signing of all pending transfers to Ethereum
+  ```
+  axelarcli tx ethereum sign-pending-transfers --from validator -y -b block
+  -> returns commandID of signed tx
+  ```
```
Comment (jcs47) [inline README.md:51]: Warn about the need to wait for the signature to be voted

**Review by jcs47 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (cgorenflo) [reply README.md:52]: done

File: README.md [hunk 3]
```diff
@@ -21,3 +21,78 @@ Optional flags:
...
+
+- Create verification json object for Axelar
+  ```
+  axelarcli q bitcoin txInfo {blockhash, e.g. 4ac9dc50dc1b952cb1efca1e634216da2f5e3a12b4a4a802ce0f6b1271876bd2} {txID:voutIdx, e.g. da5b2e8037ce4b95f40ada01c6c2cd3ccb806d0a952906130eb9b806f7887590:1}
+  -> returns json of verification info for the given outpoint (copy the escaped string)
+  ```
+- Verify the Bitcoin outpoint
+  ```
+  axelarcli tx bitcoin verifyTx {verification info (output of previous cmd), e.g. "{\"OutPoint\":{\"Hash\":\"NxF/hGLGQZ6mTNyMWkYHPJ21E+2PMb1DV/beV7R9Gpk=\",\"Index\":1},\"Amount\":\"100000000\",\"BlockHash\":\"tPqsKekDOp5lW6QUl+YwlaD/3cmbQJwuUqgiNkqloQM=\",\"Address\":\"bcrt1qrnc097fuepeyrchganj4jzl2yuf5c0fg800uenr5h0d58emztxasusnk7p\",\"Confirmations\":\"21\"}"} --from validator -y -b block
+  ```
```
Comment (jcs47) [inline README.md:98]: What about the `sign-burn-tokens` command in the ethereum module?

**Review by jcs47 — APPROVED** [review]

**Pull request merged**